rialto_airflow/dags/harvest.py

```diff
@@ -164,25 +161,23 @@ def publish(pubs_to_contribs, merge_publications):
 
     sul_pub_jsonl = sul_pub_harvest(snapshot)
 
-    dimensions_dois = dimensions_harvest_dois(snapshot)
+    dimensions_jsonl = dimensions_harvest(snapshot)
 
     openalex_jsonl = openalex_harvest(snapshot)
 
     wos_jsonl = wos_harvest(snapshot)
 
     # TODO: add dimensions_jsonl as a dependency when task is added to DAG
-    openalex_additions = fill_in_openalex(snapshot, openalex_jsonl, wos_jsonl)
+    openalex_additions = fill_in_openalex(snapshot, openalex_jsonl, wos_jsonl)  # noqa: F841
```

Comment:
the only change i made to this line was adding this comment for ruff to ignore the lack of `openalex_jsonl` usage.  no idea why it didn't fail on `main` in CI, it failed on an updated `main` for me locally after i last pulled so that i could rebase this PR.